CONTRIBUTING.md

```diff
@@ -62,7 +62,7 @@ avoid common [go style mistakes](https://github.com/golang/go/wiki/CodeReviewCom
 The documentation is published in form of a book at:
 
 - [Stable release](https://cluster-api.sigs.k8s.io)
-- [Tip of the main branch](https://tip.cluster-api.sigs.k8s.io/)
+- [Tip of the main branch](https://main.cluster-api.sigs.k8s.io/)
```

Comment:
Q: should we add link to release banch docs (https://release-0-3.cluster-api.sigs.k8s.io, https://release-0-4.cluster-api.sigs.k8s.io)?

Reply:
yes, I can do that in this PR as well

Reply:
done